src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/config/WebClientConfiguration.kt

```diff
@@ -28,12 +28,13 @@ class WebClientConfiguration(
   fun oauthApiHealthWebClient(): WebClient = WebClient.builder().baseUrl(oauthApiBaseUri).build()
 
   @Bean
-  fun nomisApiWebClient(authorizedClientManager: ReactiveOAuth2AuthorizedClientManager): WebClient {
+  fun nomisApiWebClient(authorizedClientManager: ReactiveOAuth2AuthorizedClientManager, webClientBuilder: WebClient.Builder): WebClient {
```

Comment:
We need to inject the WebClient.Builder that has been configured by Spring with the correct Codecs (specfically for date handling). We previously worked around this by using `JsonFormat(pattern = "yyyy-MM-dd")` on the request object but we can't do that now the request object is generated.